Blueprints/BlueprintDefinitions/vs2022/AnnotationsFramework/template/test/BlueprintBaseName.1.Tests/FunctionTest.cs

```diff
@@ -1,21 +1,20 @@
 using Xunit;
 using Amazon.Lambda.TestUtilities;
 
-namespace BlueprintBaseName._1.Tests
+namespace BlueprintBaseName._1.Tests;
+
+public class FunctionTest
```

Comment:
The class name and the file name should be changed to `FunctionsTest`

Reply:
Fixed